src/main/java/cs/ualberta/cmput402/tictactoe/TicTacToeGame.java

```diff
@@ -12,11 +12,16 @@
 public class TicTacToeGame {
 
     private Board board;
+    private int[] score = {0,0,0};	// X wins, O wins, Ties
```

Comment:
It is better to initial score in resetBoard.

Reply:
reset board clears the game moves if this is set there the score will not be tracked. I moved this to the constructor of TicTacToeGame instead.